docs/user-guide/deployments.md

```diff
@@ -425,15 +425,27 @@ The `.spec.template` is a [pod template](/docs/user-guide/replication-controller
 the same schema as a [Pod](/docs/user-guide/pods), except it is nested and does not have an
 `apiVersion` or `kind`.
 
+In addition to required fields for a Pod, a pod template in a Deployment must specify appropriate
+labels (see [selector](#selector)) and an appropriate restart policy.
+
+Only a [`RestartPolicy`](/docs/user-guide/pod-states/) equal to `Always` is allowed, which is the default
+if not specified.
+
 ### Replicas
 
 `.spec.replicas` is an optional field that specifies the number of desired Pods. It defaults
 to 1.
 
 ### Selector
 
-`.spec.selector` is an optional field that specifies label selectors for Pods
-targeted by this deployment. Deployment kills some of these Pods, if their
+`.spec.selector` is an optional field that specifies [label selectors](/docs/user-guide/labels/#label-selectors) for Pods
+targeted by this deployment. 
+
+If specified, `.spec.selector` must match `.spec.template.metadata.labels`, or it will
+be rejected by the API.  If `.spec.selector` is unspecified, `.spec.selector.matchLabels` will be defaulted to
+`.spec.template.metadata.labels`.
+
+Deployment kills some of these Pods, if their
```

Comment:
Deployment may kill Pods whose labels match the selector, in the cases that their